frontends/web/src/routes/settings/manage-accounts.tsx

```diff
@@ -204,53 +207,53 @@ class ManageAccounts extends Component<Props, State> {
   };
 
   public render() {
-    const { t } = this.props;
+    const { t, deviceIDs, hasAccounts } = this.props;
     const { editAccountCode, editAccountNewName, editErrorMessage } = this.state;
     const accountList = this.renderAccounts();
     return (
-      <div className="contentWithGuide">
-        <div className="container">
-          <div className="innerContainer scrollContainer">
-            <Header title={<h2>{t('manageAccounts.title')}</h2>} />
-            <div className="content">
-              <div className="columnsContainer">
-                <div className="buttons m-bottom-large m-top-large">
+      <Main>
+        <div className="hide-on-small"><Header title={<h2>{t('manageAccounts.title')}</h2>} /></div>
+        <View fullscreen={false}>
+          <ViewContent>
+            <WithSettingsTabs deviceIDs={deviceIDs} hideMobileMenu hasAccounts={hasAccounts} subPageTitle={t('manageAccounts.title')}>
+              <>
+                <div className="columnsContainer">
```

Comment:
thanks for changing to View component 👍 

since you are here... I quickly tested removing the `columnsContainer` div, it doesn't do much. It was supposed to be used in combination with `columns` + `column` iirc. It seems to add some margin-right on small screen, due to oldschool grid styles.

Reply:
True. I'll remove this div. Thanks!